install_requirements.py

```diff
@@ -111,12 +117,38 @@ def python_is_compatible():
 # The pip repository that hosts nightly torch packages.
 TORCH_NIGHTLY_URL = "https://download.pytorch.org/whl/nightly/cpu"
 
-# pip packages needed by exir.
-EXIR_REQUIREMENTS = [
-    f"torch==2.6.0.{NIGHTLY_VERSION}",
-    f"torchvision==0.20.0.{NIGHTLY_VERSION}",  # For testing.
-    "typing-extensions",
-]
+if USE_PYTORCH_NIGHTLY:
+    # pip packages needed by exir.
+    EXIR_REQUIREMENTS = [
+        f"torch==2.6.0.{NIGHTLY_VERSION}",
```

Comment:
```suggestion
        f"torch==2.6.0.{NIGHTLY_VERSION}" if USE_PYTORCH_NIGHTLY else "torch",
```
Maybe this is cleaner, so you don't have to specify other packages in both branches?

Reply:
Yeah, good idea, that looks nicer indeed